### PR TITLE
Add configurable logger, solidify workaround for JBR-5274

### PR DESCRIPTION
**Pull request by SergeevPavel**

### CODE REVIEW

**Review by manu-unter — APPROVED** [review]
LGTM!

File: skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt [hunk 1]
```diff
@@ -524,6 +530,8 @@ actual open class SkiaLayer internal constructor(
         check(isEventDispatchThread()) { "Method should be called from AWT event dispatch thread" }
         check(!isDisposed) { "SkiaLayer is disposed" }
 
+        checkContentScale()
```
Comment (igordmn) [inline skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt:533]: `update` will be only called in response on `needRedraw` and `redrawImmediately`. So, we will have the wrong density until something is changed. For example, until we hover a button.

**Review by igordmn — APPROVED** [review]
Thanks!

Logger looks good to me.

Regarding contentScale - it helps us to not miss the callback completely, but in case when we don't have animations/changes, we still have an issue. See my comment.

File: skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt (hunk 1 above)
Comment (SergeevPavel) [reply skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt:533]: I know, but it's still better than having the wrong density until a window resize. The root cause of the problem is JBR-5274, and we are going to remove this workaround when it's fixed.

Comment (igordmn) [reply skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt:533]: ok, let's merge

**Pull request merged**